src/test/user.js

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../index';
+
+chai.use(chaiHttp);
+chai.should();
+
+describe('users', () => {
+  it('User should be registered', (done) => {
+    const user = {
+      username: 'Kagabo',
+      email: 'kagabo@gmail.com',
+      password: '12345678',
+    };
+    chai.request(app)
+      .post('/api/users')
+      .send(user)
+      .end((req, res) => {
+        res.should.have.status(200);
```

Comment:
Checkout HTTP status code so that you can use the correctly. For instance, 201 is for `create` action